src/common/include/common/command.h

```diff
@@ -20,6 +20,16 @@ enum Command {
     CMD_FAN_GET = 7,
     // Set fan speeds
     CMD_FAN_SET = 8,
+
+    // Get Config by name from index.
+    CMD_GET_CONFIG_NAME = 32,
```

Comment:
Why not continue from 9?

Reply:
The main reason is to allow space so that if additional commands are needed (such as a subgroup, or config type (float / int / string / etc) are needed later on, we don't ahve an issue with mixed command ids.

If a separate command was added and everything was packed, you d then get something like
GET_CONFIG_NAME = 9
OTHER_COMMAND = 10
GET_CONFIG_TYPE = 11